Formula/catboost-cli.rb

```diff
@@ -0,0 +1,44 @@
+class CatboostCli < Formula
+  desc "Fast, scalable, high performance Gradient Boosting on Decision Trees cli tool"
+  homepage "https://catboost.ai"
+  url "https://github.com/catboost/catboost/archive/refs/tags/v1.2.1.tar.gz"
+  sha256 "b1cf4458415a639f26e953bc4e2877d68d2a711f457a725ef93cbe24363725f0"
+  license "Apache-2.0"
+  head "https://github.com/catboost/catboost.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
```

Comment:
no need to add livecheck, the default one just works out fine.
```suggestion
```

Reply:
Removed!